<?xml version='1.0'?>
<launch>
	<arg name="wp_name" default="home"/>

	<node pkg="contrail_manager" type="dispatcher" name="waypoint_dispatcher" clear_params="true" output="screen">
		<param name="action_topic" value="/emulated_uav/mavel/contrail" />
		<rosparam command="load" file="$(find egh450g1_code)/movements/$(arg wp_name).yaml"/>
	</node>
</launch>
